site/docs/components/stepped-tracker/examples.mdx

```diff
@@ -1,65 +1,74 @@
 ---
-title:
-  $ref: ./#/title
-layout: DetailComponent
-sidebar:
-  exclude: true
-data:
-  $ref: ./#/data
+  title:
+    $ref: ./#/title
+  layout: DetailComponent
+  sidebar:
+    exclude: true
+  data:
+    $ref: ./#/data
 ---
 
 <LivePreviewControls>
 
-  <LivePreview componentName="stepped-tracker" exampleName="Basic" >
+<LivePreview componentName="stepped-tracker" exampleName="Basic">
+  ## Basic
 
-## Basic
+The `SteppedTracker` component accepts `Step` components as children.
+You can change
 
-`SteppedTracker` contains multiple `TrackerStep` child components. Each `TrackerStep` indicates its current status through its icon and color. In addition, the connectors indicate the current active step and the user’s progress through the process.
+</LivePreview>
 
-You can add a label to the `TrackerStep` using the `StepLabel` component.
+## Orientation
 
-  </LivePreview>
+<LivePreview
+  componentName="stepped-tracker"
+  exampleName="OrientationHorizontal"
+>
+  ### Horizontal (default)
+</LivePreview>
 
-  <LivePreview componentName="stepped-tracker" exampleName="StageAndStatus" >
+<LivePreview componentName="stepped-tracker" exampleName="OrientationVertical">
+  ### Vertical
+</LivePreview>
 
-## Stage and Status
+<LivePreview componentName="stepped-tracker" exampleName="StageStatus">
+  ## Stage + Status
 
-`SteppedTracker` supports multiple `stage` and `status` values for each `TrackerStep` child component.
+The `Step` component changes based on the `stage` and `status` props.
+In case they are both set, `status` will take precedence over `stage`.
+These 2 props control various aspects of the component, such as:
+icon selection, description text color and track (connector) style.
 
-Available `stage` values are `"pending"` and `"completed"`. **Note:** `"active"` is not a valid value: the active step is defined by the `activeStep` prop on the `SteppedTracker` component.
+</LivePreview>
 
-Available `status` values are `"warning"` and `"error"`.
+<LivePreview componentName="stepped-tracker" exampleName="NestedSteps">
+  ## Nested Steps
 
-When stage and status are both set, the order of precedence for which icon is show is as follows: "completed" > "active" > "error|warning" > "pending"
+When using the `SteppedTracker` component in vertical orientation, you can nest `Step` components other `Step` components.
+When done so, the parent `Step` renders another `SteppedTracker` and places the nested `Steps` (children) inside of it.
+Additionally an expand button is added on the parent `Step` to expand or collapse the substeps.
+The parent Step containing the active Step child children are to have a stage of `inprogress`.
 
-  </LivePreview>
+</LivePreview>
 
-  <LivePreview componentName="stepped-tracker" exampleName="Vertical" >
+<LivePreview componentName="stepped-tracker" exampleName="Hook">
+  ## Hook
```

Comment:
These additional spaces may be the reason for right hand TOC not working

Reply:
True, navigation wasn't working before, fixed now!

Done!